src/components/Photon/photon.tsx

```diff
@@ -0,0 +1,216 @@
+import { Button, NativeSelect, Stack, TextInput } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { useCallback, useState, useEffect } from "react";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { RenderComponent } from "../UserGuide/UserGuide";
+import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { LoadingOverlayAndCancelButtonPkexec } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
+import InstallationModal from "../InstallationModal/InstallationModal";
+
+/**
+ * Represents the form values for the Photon component.
+ */
+interface FormValuesType {
+    url: string;
+    outputDir: string;
+    crawlDepth: number;
+    cookies: string;
+    userAgent: string;
+}
+
+/**
+ * The Photon component.
+ * @returns The Photon component.
+ */
+const Photon = () => {
+    // Component State Variables.
+    const [loading, setLoading] = useState(false); // State variable to indicate loading state.
+    const [output, setOutput] = useState(""); // State variable to store the output of the command execution.
+    const [pid, setPid] = useState(""); // State variable to store the process ID of the command execution.
+    const [allowSave, setAllowSave] = useState(false); // State variable to indicate if saving is allowed.
+    const [hasSaved, setHasSaved] = useState(false); // State variable to indicate if the output has been saved.
+    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
+    const [loadingModal, setLoadingModal] = useState(true); // State variable that indicates if the modal is opened.
+    const [opened, setOpened] = useState(!isCommandAvailable); // State variable to indicate loading state of the modal.
+
+    // Component Constants.
+    const title = "Photon";
+    const description =
+        "Photon is a fast crawler designed for extracting URLs, endpoints, and more from target websites.";
+    const steps =
+        "Step 1: Enter a valid target URL(eg. https://www.deakin.edu.au).\n" +
+        "Step 2: Specify additional options such as output directory, cookies, and user-agent.\n" +
+        "Step 3: Specify the crawl depth.\n" +
+        "Step 4: Click Launch Crawler to start Photon.\n" +
```

Comment:
It should be Start "+ title +"
```suggestion
        "Step 4: Click Start "+ title +" to start Photon.\n" +
```